modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -78,7 +81,7 @@ You can export the view of your network showing on your canvas to a PNG file.
 * If you customized your view with filters, the customized network topology is exported.
 * If you selected metrics or durations, these settings are reflected in the export.
 
-To export a PNG view of your network, click *Export as PNG* in the side panel.
+To export a PNG view of your network, click *Export as Image* next to the zoom to fit.
```

Comment:
click the "Export as Image" button next to the zoom controls in the bottom right.

Reply:
Accepted.